### ISSUE TITLE
no widget shows and no error message

### ISSUE THREAD

**Issue by babaq** [JuliaGizmos/Interact.jl#182]
Hi

I just found that simple widget doesn't show anything, and no error given. tested on `Julia 0.6` Win10, latest `Interact 0.6.2`, `Ijulia 1.6.1` using default `conda` python distribution.

```Julia
using Interact
slider(1:10)
```

**Comment by JobJob** [comment]
Try 
```
Pkg.checkout("Interact")
Pkg.build("Interact")
```

A new version should be tagged shortly

**Comment by korsbo** [comment]
I just tried this and it did not work with ipywidgets 7.0.1
It did however work after I downgraded ipywidgets to 6.0.0

**Comment by JobJob** [comment]
Can you paste the output of Pkg.build("Interact") please

**Comment by JobJob** [comment]
Thanks or checking that out btw @korsbo. Your `Pkg.build("Interact")` output is of interest too, make sure you run it after changing your IPywidgets version, and everything should be roses.

As you might expect, everything works fine for me (with IPywidgets 6.0, 7.0 upgrading downgrading - as long as I run `Pkg.build("Interact")`, and refresh the browser page after), but there may be an issue with how we are getting the python version that is running Jupyter.

**Comment by korsbo** [comment]
I will be demoing my code to my professor in a few hours, so now that I've got it all working I do not want to touch it. I did, however, take a screenpic when I was tinkering this morning: 
<img width="714" alt="screen shot 2017-09-18 at 10 18 21" src="https://user-images.githubusercontent.com/16306680/30540681-126ac482-9c70-11e7-9c50-dff66bcfc2e8.png">

Here, I did have ipywidgets 7.0.1 installed (regardless of what the build output claims). 
When stuff did not work, I manually checked out master and merged with a newly fetched origin/master to make sure that I really had the code that I thought. This did not help (I did restart the kernel and re-built Interact).

**Comment by babaq** [comment]
I checked out the lastest release and build, still not working.
@JobJob the output did show that it can't find the jupyter python version on windows.

![image](https://user-images.githubusercontent.com/92771/30561163-f2e8739e-9c6e-11e7-879d-e5278d8928ea.png)

**Comment by JobJob** [comment]
Ok thanks. Workaround for now is create a file `(path to Interact)/deps/ipywidgets_version` and save `6.0.0` in it, or if you have IPywidgets version >= 7.0.0 save `7.0.0` in it. Then restart the  IJulia kernel, and refresh the browser.

A better solution is forthcoming.

**Comment by babaq** [comment]
@JobJob it worked, thanks. looking forward for the solution.

**Comment by JobJob** [comment]
Cheers, just gonna leave this open for now, so it's more easily seen.

**Comment by SimonEnsemble** [comment]
I am having the same problem: see #199.

I have IPywidgets 7.05 installed via pip, but then `Pkg.build("Interact")` keeps finding v6.0.0.

I changed `~/.julia/v0.6/Interact/deps/ipywidgets_version` to say `7.0.5`, but then it *still* keeps finding v6.0.0 upon buliding! And, when I check the file again, it says `6.0.0`, as if it is being overwritten during the build process.

**Comment by babaq** [comment]
@SimonEnsemble did you tried `7.0.0` in `~/.julia/v0.6/Interact/deps/ipywidgets_version`, may be it doesn't care minor version numbers?

**Comment by SimonEnsemble** [comment]
@babaq thanks, I tried but `7.0.0` instead of `7.0.5` did not solve the problem.

**Comment by babaq** [comment]
In python prompt,
```python
import ipywidgets
ipywidgets.__version__
```
can show your ipython verison number, if it is 7.0.5 and setting 7.0.0 doesn't work, then maybe the lastest ipywidgets are yet to be supported.

**Comment by SimonEnsemble** [comment]
@JobJob Wicked, that fixed it ha ha, thanks!

**Comment by stevengj** [comment]
I just had this problem today, and I tracked down the source.  The issue arises when you have multiple `jupyter` installations on your machine, and IJulia (and hence) Interact are configured with a different one than the `jupyter` you are actually running.

In particular, the issue arises when `IJulia.jupyter` is different from the `jupyter` executable you are actually running on your machine — then the Interact build script gets confused and looks at `ipywidgets` from the wrong Python version.

Look at `using IJulia; IJulia.jupyter`.  If this is not the path of the `jupyter` executable you want to run on your machine, then do:
```jl
ENV["JUPYTER"] = "/path/to/jupyter"
Pkg.build("IJulia")
Pkg.build("Interact")
```
where `/path/to/jupyter` is the pathname of the `jupyter` program that you intend to use.

Now that Interact.jl is actually finding the correct ipywidgets version (7.0.5), it is working again for me, so I think ipywidgets 7 is fine.

**Comment by JobJob** [comment]
Thanks @stevengj !

**Comment by JobJob** [comment]
This should be fixed as of #209
